web/src/ui/App.tsx

```diff
@@ -12,36 +12,37 @@ import uuid from "uuid";
 import { Icon, ICONS, IconType, WALL_ICON } from "./icons";
 import FloorTokenSheet from "./FloorTokenSheet";
 import Board from "./Board";
+import SearchDialog from "./SearchDialog";
 
 const dragSnapToGrid = (x: number) =>
-  Math.round(x / GRID_SIZE_PX) * GRID_SIZE_PX;
```

Comment:
Last time we talked about this, I remember we liked round better than floor for dragging, because tokens would snap to the closest square, rather than always go to the top-left corner. Why is this being changed now?

Reply:
This used to be calculated from the top left of the token, so it made more sense to round. Now it's calculated from the mouse position, so floor makes more sense